src/Education/IEducation.cs

```diff
@@ -61,8 +62,6 @@ public Education(IToolWindowService toolWindowService, IRuleMetadataProvider rul
             this.ruleMetadataProvider = ruleMetadataProvider;
             this.logger = logger;
             this.threadHandling = threadHandling;
-
-            ruleHelpToolWindow = toolWindowService.GetToolWindow<RuleHelpToolWindow, IRuleHelpToolWindow>();
```

Comment:
We should be on the UI thread when making this since it indirectly calls through to `IVsUIShell`.

We have an `IThreadHandling` instance so we could switch to the UI thread here. However, we already have a switch to the UI thread in the `ShowRuleHelp` method, so we can just refer the work until then i.e. lazily-fetch the tool window.